### ISSUE TITLE
1.11.0 Breaks multiple mongo connection url

### ISSUE THREAD

**Issue by StribBrentDoll** [strongloop/loopback-connector-mongodb#146]
It looks like the work around in PR #142 breaks our use case for multiple servers.

Our datasources.json looks like:

```json
{
  "db": {
    "connector": "mongodb",
    "url": "mongodb://my_user:passw0rd@primary.example.com:53069,secondary.example.com:53069/my_db?readPreference=secondary"
  }
}
```

The url `connectToMongoDB` tries to connect to ends up being:

`mongodb://@primary.example.com:53069,secondary.example.com:53069/my_db?readPreference=secondary`

The app then times out on any connections to mongo

**Comment by seeker89** [comment]
+1 I'm also seeing this behaviour.
Spent last 2 hours trying to see what happened to my firewalls...

**Comment by raymondfeng** [comment]
Good catch. Should be fixed in 1.11.1 now.
